src/components/ErrorBoundary/index.tsx

```diff
@@ -33,179 +31,78 @@ const CodeBlockWrapper = styled.div`
   color: ${({ theme }) => theme.deprecated_text1};
 `
 
-const LinkWrapper = styled.div`
-  color: ${({ theme }) => theme.deprecated_blue1};
+const Padding = styled.div`
   padding: 6px 24px;
 `
 
-const SomethingWentWrongWrapper = styled.div`
-  padding: 6px 24px;
-`
-
-type ErrorBoundaryState = {
-  error: Error | null
+const Fallback = ({ error }: { error: Error }) => {
+  return (
+    <FallbackWrapper>
+      <BodyWrapper>
+        <AutoColumn gap="md">
+          <Padding>
+            <ThemedText.DeprecatedLabel fontSize={24} fontWeight={600}>
+              <Trans>Something went wrong</Trans>
+            </ThemedText.DeprecatedLabel>
+          </Padding>
+          <CodeBlockWrapper>
+            <code>
+              <ThemedText.DeprecatedMain fontSize={10}>{error.stack}</ThemedText.DeprecatedMain>
+            </code>
+          </CodeBlockWrapper>
+          <AutoRow>
+            <Padding>
+              <ExternalLink id="get-support-on-discord" href="https://discord.gg/FCfyBSbCU5" target="_blank">
+                <ThemedText.DeprecatedLink fontSize={16} color="deprecated_blue1">
+                  <Trans>Get support on Discord</Trans>
+                  <span>↗</span>
+                </ThemedText.DeprecatedLink>
+              </ExternalLink>
+            </Padding>
+          </AutoRow>
+        </AutoColumn>
+      </BodyWrapper>
+    </FallbackWrapper>
+  )
 }
 
-const IS_UNISWAP = window.location.hostname === 'app.uniswap.org'
```

Comment:
This is important - we don't want to bother tracking forks, or usages through IPFS.

Reply:
Correct, however, we don't do that while initialising Uniswap Analytics in `App.tsx`. Does that service have special handling for hostname already included?

Reply:
I think we can safely ignore this on a code-level (since forks can remove that code too) and instead, configure Sentry to only accept requests coming from `app.uniswap.org`. We would have to do it anyway, since the URL for sending events is public.

This can be done under Project Settings -> Client Security.

Reply:
<img width="925" alt="Screenshot 2022-12-02 at 02 39 15" src="https://user-images.githubusercontent.com/2464966/205174097-46f83603-ba4a-4633-9a5f-017f66763329.png">

Here's a screenshot. Purposefully set to `*` in case we have staging or other environments, but we can narrow it down to just `app` too.